includes/WC_Payments_Compatibility.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ * Class for WooPayments compatibility.
+ */
+
+namespace WooCommerce\Square;
+
+use WooCommerce\Square\Handlers\Product;
+
+/**
+ * WC_Payments_Compatibility Class
+ *
+ * @version x.x.x
+ */
+class WC_Payments_Compatibility {
+	public function init() {
```

Comment:
There's a missing function comment here.
```suggestion
        
        /**
	 * Initialize the WooPayments Compatibility class.
	 *
	 * @since x.x.x
	 */
	public function init() {
```